doc/developer/guide-testing.md

```diff
@@ -257,11 +259,65 @@ provide most of the coverage, but it's worth adding some (*very*) basic
 testdrive tests (e.g., `> SELECT DATE '1999-01-01'`) to ensure that our pgwire
 implementation is properly serializing dates.
 
-Like the [Unit Tests](#unitintegration-tests), many testdrive tests require
-Zookeeper, Kafka, and the Confluent Schema Registry.
+#### Usage via mzcompose/Docker
+
+The easiest way to run testdrive tests is via mzcompose.
+
+Testdrive comes with a Docker Compose environment that can set all
+of this up for you, including re-building and running the testdrive
+and materialized binaries if you have changed them locally. From the
+`test/testdrive` directory:
```

Comment:
The first sentence in this paragraph no longer makes sense given the reordering, since "all of this" hasn't been defined yet. Can we rephrase to something like this?

> The mzcompose configuration will automatically set up all of testdrive's dependencies, including Zookeeper, Kafka, the Confluent Schema Registry, and mock versions of AWS S3 and Kinesis.